Makefile

```diff
@@ -130,13 +133,28 @@ push-mmlogicapi-image: build-mmlogicapi-image
 push-backendclient-image: build-backendclient-image
 	docker push $(REGISTRY)/openmatch-backendclient:$(TAG)
 
+push-mmf-cs-mmlogic-simple-image: build-mmf-cs-mmlogic-simple-image
+	docker push $(REGISTRY)/openmatch-mmf-cs-mmlogic-simple:$(TAG)
```

Comment:
A naming suggestion - why do we have mmlogic in the example function names?  - a simple cs mmf example can just be mmf-cs-simple?

Reply:
I ll rename these in a separate CL! for now - checkin.